src/test/java/com/wikia/webdriver/common/templates/NewTestTemplateCore.java

```diff
@@ -66,11 +68,18 @@ protected void startBrowser() {
     driver =
         registerDriverListener(NewDriverProvider.getDriverInstanceForBrowser(Configuration
             .getBrowser()));
+    try {
+      Thread.sleep(1000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1148)